client/jetbrains/webview/src/BUILD.bazel

```diff
@@ -0,0 +1,16 @@
+load("//client:tools.bzl", "module_style_typings")
+load("@aspect_rules_js//js:defs.bzl", "js_library")
+
+module_style_typings(deps = ["//client/wildcard/src"])
+
+js_library(
```

Comment:
We need to drop these because they copy all the `.ts`  to the outdir, the `.ts` should only be inputs to the `generate_graphql_operations`. Does `filegroup` work for cases like this where these files are consumed in a different BUILD (so you can't simply put them into the `generate_graphql_operations(srcs)).

Reply:
As discussed, need to copy these to bin. Using `js_library` more idiomatic for rules_js than using `copy_to_bin`.